src/services/functionAppService.test.ts

```diff
@@ -297,29 +303,18 @@ describe("Function App Service", () => {
         Authorization: `Bearer ${(await variables["azureCredentials"].getToken()).accessToken}`,
         Accept: "*/*",
         ContentType: "application/octet-stream",
+        "User-Agent": "serverless_framework",
       }
-    }, slsService["artifact"]);
-    const expectedArtifactName = service.getDeploymentName().replace("rg-deployment", "artifact");
-    expect((AzureBlobStorageService.prototype as any).uploadFile).toBeCalledWith(
-      slsService["artifact"],
-      configConstants.deploymentConfig.container,
-      `${expectedArtifactName}.zip`,
-    )
-    const uploadCall = ((AzureBlobStorageService.prototype as any).uploadFile).mock.calls[0];
-    expect(uploadCall[2]).toMatch(/.*-t([0-9]+)/)
+    }, "app.zip");
   });
 
-  it("uploads functions to function app and blob storage with default naming convention", async () => {
-    const scmDomain = app.enabledHostNames.find((hostname) => hostname.endsWith("scm.azurewebsites.net"));
-    const expectedUploadUrl = `https://${scmDomain}/api/zipdeploy/`;
-
+  it("publishes functions and uploads to blob storage with default naming convention", async () => {
     const sls = MockFactory.createTestServerless();
     delete sls.service["artifact"]
     const service = createService(sls);
     await service.uploadFunctions(app);
-
     const defaultArtifact = path.join(".serverless", `${sls.service.getServiceName()}.zip`);
-
+    const expectedUploadUrl = `https://${app.enabledHostNames[1]}/api/zipdeploy`;
```

Comment:
nit: rename to `expectedPublishUrl` since it could be confuse with blob storage upload